KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/KeywordSearch.java

```diff
@@ -126,7 +126,7 @@ public void propertyChange(PropertyChangeEvent evt) {
                     } catch (Exception ex) {
                         String caseName = closedCase.getName();
                         logger.log(Level.SEVERE, String.format("Failed to close core for %s", caseName), ex);
-                        MessageNotifyUtil.Notify.error(NbBundle.getMessage(KeywordSearch.class, "KeywordSearch.closeCore.notification..msg"), ex.getCause().getMessage());
+                        MessageNotifyUtil.Notify.error(NbBundle.getMessage(KeywordSearch.class, "KeywordSearch.closeCore.notification.msg"), ex.getMessage());
```

Comment:
There is not always a cause, which leads to an NPE. I decided to standardize the scheme rather than have a branch on whether or not getCause() returned null. Especially given that we decided to hide the raw exceptions with our own class. The log should show the gory details.